doc/sphinx/using/libraries/funind.rst

```diff
@@ -1,6 +1,18 @@
 Functional induction
 ====================
 
+.. note::
+
+   The functional induction (FunInd) plugin is considered legacy. For new
+   code and new projects, the recommended replacement is `Equations
+   <http://mattam82.github.io/Coq-Equations/>`_, a more powerful plugin
+   available through the Coq Platform, which provides most features from
+   FunInd as well as several others. Refer to the `Equations documentation
```

Comment:
```suggestion
   code and new projects, the recommended replacement is `Equations
   <http://mattam82.github.io/Coq-Equations/>`_, a more powerful plugin
   that provides most of FunInd's features and more.  It can be installed
   through the Coq Platform. Refer to the `Equations documentation
```

Hmm, you like the passive voice better than "we recommend"?  In academic papers, one would avoid the first person, but among friends it's nice to use the first person (plural).

Reply:
It's a writing habit, I guess, but I don't mind using "we".